server_benchmark.py

```diff
@@ -15,6 +15,8 @@
 import ga4gh.protocol as protocol
 import ga4gh.datamodel.variants as variants
 
+from guppy import hpy
```

Comment:
I tend to prefer `import guppy` and use `guppy.hpy` (see the style PR). However, this might  be the pattern required for this to work, so feel free to ignore in this case.